docs/js-api.rst

```diff
@@ -11,6 +11,17 @@ API exposed by the wrapper.
 First a warning: **the JavaScript API is experimental**.  It may
 change incompatibly even in minor versions of the compiler.
 
+A Futhark program ``futlib.fut`` compiled with a WASM backend as a library
+with the ``--library`` command line option produces five files: ``futlib.c``,
+``futlib.h``, ``futlib.js``, ``futlib.wasm``, and ``futlib.class.js`` .  The
+``.c`` and ``.h`` are the implementation and header C file generated by the
+Futhark compiler. ``futlib.class.js`` is the JavaScript code generated to wrap
+the Emscripten generated files ``futlib.js`` and ``futlib.wasm`` with a friendlier
+API.
```

Comment:
Does the `futlib.wasm` file have to be present at run-time?  If so, that should be mentioned.  If not, that should be clarified.

Reply:
Yes it does. I will add some more text saying that `futlib.js` and `futlib.wasm` must be present during runtime